docs/scos/dev/guidelines/keeping-a-project-upgradable/supported-extension-scenarios/modules-configuration.md

```diff
@@ -12,102 +12,102 @@ related:
+use Spryker\Zed\Customer\CustomerConfig as SprykerCustomerConfig;
 
-class ConfigurableBundleStorageConfig extends SprykerConfigurableBundleStorageConfig
+class CustomerConfig extends SprykerCustomerConfig
 {
     ...
-    public function getConfigurableBundleTemplateSynchronizationPoolName(): string
+    public function getCustomerPasswordSequenceLimit(): ?int
     {
-        return SynchronizationConfig::DEFAULT_SYNCHRONIZATION_POOL_NAME;
+        return 3;
     }
 }
 ```
 
-Code example 1.1.3: Method returns string with constant concatenation
+The following is an example of how to have the method return a string:
 ```php
-use Spryker\Zed\Development\DevelopmentConfig as SprykerDevelopmentConfig;
+use Spryker\Client\RabbitMq\RabbitMqConfig as SprykerRabbitMqConfig;
 
-class DevelopmentConfig extends SprykerDevelopmentConfig
+class RabbitMqConfig extends SprykerRabbitMqConfig
 {
     ...
-    public function getCodingStandard(): string
+    protected function getDefaultBoundQueueNamePrefix(): string
     {
-        return APPLICATION_ROOT_DIR . DIRECTORY_SEPARATOR . 'phpcs.xml';
+        return 'error';
     }
 }
 ```
 
-Code example 1.1.4: Method returns boolean
+Manifests also support the usage of constants and have built in basic support for constant concatenation.
+
+The following is an example of how to have the method return a string with a constant:
+
 ```php
-use SprykerShop\Yves\ProductReplacementForWidget\ProductReplacementForWidgetConfig as SprykerShopProductReplacementForWidgetConfig;
+use Pyz\Zed\Synchronization\SynchronizationConfig;
+use Spryker\Zed\ConfigurableBundleStorage\ConfigurableBundleStorageConfig as SprykerConfigurableBundleStorageConfig;
 
-class ProductReplacementForWidgetConfig extends SprykerShopProductReplacementForWidgetConfig
+class ConfigurableBundleStorageConfig extends SprykerConfigurableBundleStorageConfig
 {
     ...
-    public function isProductReplacementFilterActive(): bool
+    public function getConfigurableBundleTemplateSynchronizationPoolName(): string
     {
-        return true;
+        return SynchronizationConfig::DEFAULT_SYNCHRONIZATION_POOL_NAME;
     }
 }
 ```
 
-Code example 1.1.5: Method returns int
+The following is an example of how to have the method return a string with constant concatenation:
+
 ```php
-use Spryker\Zed\Customer\CustomerConfig as SprykerCustomerConfig;
+use Spryker\Zed\Development\DevelopmentConfig as SprykerDevelopmentConfig;
 
-class CustomerConfig extends SprykerCustomerConfig
+class DevelopmentConfig extends SprykerDevelopmentConfig
 {
     ...
-    public function getCustomerPasswordSequenceLimit(): ?int
+    public function getCodingStandard(): string
     {
-        return 3;
+        return APPLICATION_ROOT_DIR . DIRECTORY_SEPARATOR . 'phpcs.xml';
     }
 }
 ```
 
-## 1.2. Array as return
+There's a multitude of ways to view arrays as the return. Manifests support the following array types:
```

Comment:
Please consider still leaving the title "Array as a return", as the sentence "There's a multitude of ways to view arrays as the return. Manifests support the following array types:" looks very torn away from the previous section